job_application/views.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+from django.shortcuts import render, get_object_or_404
+from jobboard.models import Job
+from jobboard.views import check_if_student_already_applied
+from .models import Application
+from student.models import Student
+
+
+def apply_success(request):
+    return render(request, 'apply_success.html')
```

Comment:
I can't find where you're using this function. If it's not in use than delete it

Reply:
I think you are right. Will be next to the computer on 14:00, and will delete it